src/utils/create-image-mesh.ts

```diff
@@ -1,20 +1,27 @@
 import { createPlaneBufferGeometry } from "../utils/three-utils";
 import { errorTexture } from "../utils/error-texture";
 import { Layers } from "../camera-layers";
+import { Texture } from "three";
 
-export const AlphaMode = Object.freeze({
-  Blend: "blend",
-  Mask: "mask",
-  Opaque: "opaque"
-});
+export enum AlphaMode {
+  Opaque = 0,
+  Blend = 1,
+  Mask = 2
+}
+
+export const enum ALPHA_MODE {
+  OPAQUE = "opaque",
+  BLEND = "blend",
+  MASK = "mask"
+}
```

Comment:
1. Similar to `ProjectionMode`, maybe having `AlphaMode` and `ALPHA_MODE` enums may be confusing because the names are similar? How about renaming the latter to `AlphaModeName` or something like that?
2. Not a blocking issue but a future suggestion: There might be another properer place to define these enums, eg: `src/utils/alpha-mode.ts` similar to `src/utils/projection-mode.ts`.

Reply:
Updated